USER_MANUAL.md

```diff
@@ -0,0 +1,36 @@
+# SpatialOS GDK for Unreal Test Gyms Project 
+
+#### What is a gym
+*A gym is a level containing a minimal number of actors, with simple behaviours, necessary to demonstrate some peice of functionality in the Spatial GDK for Unreal.
+*We use gyms to helps quickly, visualy identifiy if some feature is working.
+
+#### Project layout
+```
+Content
+|
+|__Actors
+|
+|__Characters
+|
+|__GameModes
+|
+|__Geometry
+|
+|__Maps
+|
+|__Meshes
```

Comment:
What's the difference between Geometry and Meshes?

Reply:
They are just folders we didn't delete. So whatever the blank project puts in them. Presumably one is physics and one is rendering?

Reply:
Discussed offline. Will get rid of the geometry folder